plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveDistributedJoinQueries.java

```diff
@@ -41,6 +41,8 @@ protected QueryRunner createQueryRunner()
     {
         verify(new DynamicFilterConfig().isEnableDynamicFiltering(), "this class assumes dynamic filtering is enabled by default");
         return HiveQueryRunner.builder()
+                .addExtraProperty("retry-policy", "NONE") // See TestHiveFaultTolerantExecutionJoinQueries for tests with task retries enabled
```

Comment:
It's not enabled by default, still the same defaults, I just disabled it here explicitly to make sure that it always runs without retry policy since there is already a TestHiveFaultTolerantExecutionJoinQueries.
I can remove it if it looks confusing.